<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Patient</class>
 <widget class="QWidget" name="Add_Patient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add_Patient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label_head">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#00007f;&quot;&gt;Add New Patient: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_firstName">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>First Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_firstName">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_lastName">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>Last Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_lastName">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_preImage">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>Pre Image: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_preImage">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_PreImage">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_postImage">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>Post Image: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_postImage">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_PostImage">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_eyepatch">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>Eyepatch: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_leftEyePatch">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
           <property name="text">
            <string>Left Eyepatch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_rightEyePatch">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
           <property name="text">
            <string>Right Eyepatch</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_hospital">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>Hospital: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_hospital">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;
color: #0000ff;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_state">
           <property name="styleSheet">
            <string notr="true">font-size: 32px;
</string>
           </property>
           <property name="text">
            <string>State: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_state">
           <property name="styleSheet">
            <string notr="true">font-size: 28px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>180</number>
         </property>
         <property name="leftMargin">
          <number>60</number>
         </property>
         <property name="rightMargin">
          <number>60</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_ok">
           <property name="styleSheet">
            <string notr="true">font-size: 26px;</string>
           </property>
           <property name="text">
            <string>Analyze-&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cancel">
           <property name="styleSheet">
            <string notr="true">font-size: 26px;</string>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
